modules/tensor_mechanics/src/materials/ComputeInterfaceStress.C

```diff
@@ -18,51 +18,75 @@ validParams<ComputeInterfaceStress>()
   params.addClassDescription(
       "Stress in the plane of an interface defined by the gradient of an order parameter");
   params.addCoupledVar("v",
-                       "Order parameter that defines the interface. The interface is the region "
+                       "Order parameters that defines the interface. The interface is the region "
                        "where the gradient of this order parameter is non-zero.");
-  params.addRequiredParam<Real>("stress", "Planar stress");
-  params.addRangeCheckedParam<Real>("op_range",
-                                    1.0,
-                                    "op_range > 0.0",
-                                    "Range over which order parameters change across an "
-                                    "interface. By default order parameters are assumed to "
-                                    "vary from 0 to 1");
+  params.addRequiredParam<std::vector<Real>>("stress", "Planar stress");
+  params.addRangeCheckedParam<std::vector<Real>>(
+      "op_range",
+      {1.0},
+      "op_range > 0.0",
+      "Range over which order parameters change across an "
+      "interface. By default order parameters are assumed to "
+      "vary from 0 to 1");
   params.addParam<MaterialPropertyName>(
       "planar_stress_name", "extra_stress", "Material property name for the planar stress");
   return params;
 }
 
 ComputeInterfaceStress::ComputeInterfaceStress(const InputParameters & parameters)
   : Material(parameters),
-    _grad_v(coupledGradient("v")),
-    _stress(getParam<Real>("stress") / getParam<Real>("op_range")),
+    _nvar(coupledComponents("v")),
+    _grad_v(_nvar),
+    _op_range(getParam<std::vector<Real>>("op_range")),
+    _stress(getParam<std::vector<Real>>("stress")),
     _planar_stress(
         declareProperty<RankTwoTensor>(getParam<MaterialPropertyName>("planar_stress_name")))
 {
+  if (_stress.size() == 1)
+    _stress.assign(_nvar, _stress[0]);
+  if (_stress.size() != _nvar)
+    mooseError("Supply either one single stress or one per order parameter");
```

Comment:
Strong preference: error messages tell me which input block / MOOSE class is failing.

Reply:
Well, you are in for a treat, thanks to @rwcarlsen's work on the parser. Commit coming up...